Wiki/de/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Gridcoin Wiki
+layout: wiki-de
+---
+
+## Gridcoin Einführung
+
+Gridcoin (Ticker: GRC) ist eine dezentrale, mathematische,
+open source Kryptowährung (digitaler Vermögenswert). 
+Gridcoin ist, neben seiner Energieeffizienz, auch die
+erste und derzeit einzige Kryptowährung, die Einzelpersonen
+für wissenschafftliche Beiträge belohnt und peer-to-peer 
+Transaktionen kryptographisch vollzieht, ohne dabei auf 
+eine zentrale Instanz zur Verteilung der Belohnungen angewiesen 
+zu sein. Gridcoin ist das erste Blockchain Protokoll welches 
+einen funktionierenden Algorithmus zum Nachweis und Belohnung von 
+geleisteter [BOINC](https://en.wikipedia.org/wiki/Berkeley_Open_Infrastructure_for_Network_Computing)
+(Berkeley Open Infrastructure for Network Computing) Rechenarbeit,
+wobei es sich um quasi nahezu jeglichen Computerprozess handeln 
+kann, liefert.
+
+Durch die zur Verfügung gestellte Computerkapazitäten und den damit 
+ausgeführte Rechenprozessen bietet Gridcoin der Menschheit einen Nutzen. 
+Es gibt keine Einschränkungen für ein einzelnes BOINC Projekt, was zur
+Folge hat, dass so gut wie jeder Computervorgang belohnt werden kann.
+
+Einer Nachricht auf [CryptocurrencyTalk](https://cryptocurrencytalk.com/topic/1416-gridcoin-grc-information-classic-in-retirement/) zufolge, wurde Gridcoin am 16. Oktober 2013 von 
+Rob Halförd gegründet.
+
+Die Coin entschädigt die "Miner" für ihren Beitrag in [BOINC](boinc "wikilink")
+Projekten, welche Fortschritte in Bereichen wie Medizin, Biologie, 
+Mathematik, Klimaforschung, Partikel- oder Astrophysik erzielen können
+und somit erfolgreiche wissenschaftliche Arbeit für die Menschheit zur Folge 
+haben können.
+
+Gridcoin entlohnt die Nutzer für die Bereitstellung ihrer Computer, 
+indem es die Proof-of-Stake Methode anwendet. Diese ist im Gegensatz 
+zum rechenintensiven und somit hitzeerzeugenden Proof-of Work Algorithmus
+deutlich ressourcenschonender und energieeffizient. Gridcoin unterscheidet 
+sich von anderen wissenschaftlichen Kryptowährungen durch seine Fähigkeit, 
+viele verschiedene Arten von Projekten unterstützen zu können. Die Zahl derer 
+wird im Grunde nur durch die momentan aktiven BOINC Projekte und durch die 
+"project whitelist", über welche die Community abstimmt, begrenzt.
+
+Es gab in der Vergangenheit einen Wechsel vom originalen [Gridcoin-Classic](gridcoin-classic "wikilink")
+zu "Gridcoin-Research", der eigentlichen Gridcoin. Gridcoin-Classic nutzte einen 
+Proof-of-Work Hybriden, wodurch die Computer die halbe Rechenzeit mit "Hashing" beschäftigte,
+während sie die andere Hälfte ihren Beitrag zur Wissenschaft leisteten; Zwar eine
+Verbesserung zu Bitcoin, es wurden aber immer noch Computerressourcen vergeudet.
```

Comment:
```suggestion
Proof-of-Work Hybriden, wodurch die Computer die Hälfte ihrer Rechenzeit ihren Beitrag zur Wissenschaft leisteten 
 und die andere Hälfte mit "Hashing" verbrachten. Zwar eine Verbesserung zu Bitcoin, aber es wurden immer noch Computerressourcen vergeudet.
```

Reply:
Here as well. I'd write Proof of Work instead of Proof-of-Work.

Reply:
```suggestion
Proof of Work Hybriden, wodurch die Computer die halbe Rechenzeit mit "Hashing" beschäftigte,
während sie die andere Hälfte ihren Beitrag zur Wissenschaft leisteten; Zwar eine
Verbesserung zu Bitcoin, es wurden aber immer noch Computerressourcen vergeudet.
```